lib/order.rb

```diff
@@ -0,0 +1,50 @@
+
+class Order
+  MENU = { Pizza: 5,
+           Pasta: 4,
+           Steak: 6,
+           Cod: 5 }
+
+  attr_reader :menu, :ordered_dishes
+
+  def initialize menu = MENU, twilio_sms
+    @menu = menu
+    @ordered_dishes = {}
+    @total = 0
+    @twilio_sms = twilio_sms
+    @number = '+44**********'
+  end
+
+  def show_menu
```

Comment:
'show' implies that this somehow creates a view rather than just returning a string.